updates/1/updates.sql

```diff
@@ -1 +1,28 @@
-FILL ME IN
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 1604022038, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
```

Comment:
All cryptocompare entries need to be uppercase. You can already see that in all_assets.json.

It was a limitation in their API if I recall correctly